styles.css

```diff
@@ -395,3 +229,264 @@ body {
   font-weight: 500;
 }
 
+}
+
+/* MID-SIZE LAYOUT */
+@media all and (min-width: 700px) {
+  html {
+    height: 100%;
+    width: 100%;
+  }
+  
+  body {
+    margin: 0;
+    height: 100%;
+    width: 100%;
+    background: #EFEFEF;
+    font-family: 'Libre Franklin', sans-serif;
+  }
+  
+  .wrapper {
+    height: 100%;
+    width: 100%;
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: 20% 80%;
+    grid-template-areas: 
+      "nav"
+      "main";
+  }
+  
+  .nav-bar {
+    grid-area: nav;
+    background: #053353;
+    color: #EFEFEF;
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  .nav-list {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    font-size: .9em;
+    list-style-type: none;
```

Comment:
This rule is already applied to the `html` selector so this is a duplicate. I think that this is the better spot for the rule to live, though, so you could remove the higher level one instead!